ob_intern/Controller/Short Video List/cell/ShortVideoListCollectionViewCell.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CollectionViewCell.swift
+//  ob_intern
+//
+//  Created by Soft Liampisan on 7/6/2566 BE.
+//
+
+import UIKit
+import SDWebImage
+
+class ShortVideoListCollectionViewCell: UICollectionViewCell {
+
+    @IBOutlet weak var imageViewVideo: UIImageView!
+    @IBOutlet weak var labelViews: UILabel!
```

Comment:
```suggestion
    @IBOutlet weak var labelViews: UILabel!
```
เปลี่ยนเป็น labelTitle

Reply:
เปลี่ยนเป็นlabelViewCount